server/reflector/views/transcripts.py

```diff
@@ -201,7 +201,7 @@ async def transcripts_create(
     user_id = user["sub"] if user else None
     return await transcripts_controller.add(
         info.name,
-        source_kind=SourceKind.LIVE,
+        source_kind=SourceKind.FILE,
```

Comment:
I don't think it works, because the frontend also create a transcript and use it live. You need to dig further